docs/ops/detection/Proposal_1.md

```diff
@@ -31,16 +31,14 @@
   * **Required**: *yes*
 
 * *pre_nms_topn*
-
-  * **Description**: *pre_nms_topn* is the number of bounding boxes before the NMS operation. For example, *pre_nms_topn* equal to 15 means that the minimum box size is 15.
+  * **Description**: *pre_nms_topn* is the number of bounding boxes before the NMS operation. For example, *pre_nms_topn* equal to 15 means to take top 15 boxes with the highest scores.
```

Comment:
Please, update 'logits' -> 'box deltas' in this file also.

Reply:
Changed.